src/Caching/Storages/SQLiteJournal.php

```diff
@@ -41,10 +42,9 @@ public function __construct($path)
 				key BLOB NOT NULL,
 				priority INT NOT NULL
 			);
-			CREATE INDEX IF NOT EXISTS idx_tags_key ON tags(key);
```

Comment:
And [this delete](https://github.com/nette/caching/blob/08bbedcb9569103126325023c67a3d5e6ea7d4bf/src/Caching/Storages/SQLiteJournal.php#L64)?